src/subcommand/migrate.rs

```diff
@@ -7,10 +7,13 @@ use structopt::StructOpt;
 #[structopt(about = "Migrate from z database")]
 pub struct Migrate {
     path: String,
+
+    #[structopt(long, help = "Merge conflicting entries")]
```

Comment:
I think a help message of "Merge entries into existing database" would be more accurate.

Reply:
Agreed. Thanks!